gene/etl/ensembl.py

```diff
@@ -53,13 +52,13 @@ def _download_data(self):
                                self._fn)
             logger.info('Successfully downloaded Ensembl data file.')
 
-    def _extract_data(self, *args, **kwargs):
+    def _extract_data(self, ens_dir=PROJECT_ROOT / 'data' / 'ensembl',
```

Comment:
do you still need the default `ens_dir` value here as well as above (line 41)? I think in the other normalizers we define them once as class variables in __init__. I'm not sure if that messes up how you've set up the mock tests, but just a thought.

Reply:
Nah, I was in a rush to try and get mock to work so just left it. Will update the source classes to include data dir.